pkg/model/utils.go

```diff
@@ -72,3 +72,24 @@ func GetValidNameFromFolder(folder string) (string, error) {
 	log.Infof("autogenerated name: %s", name)
 	return name, nil
 }
+
+//ExpandEnvWithDefaults expands the envvaars in the given string
+func ExpandEnvWithDefaults(value string) string {
```

Comment:
will this also cover the case where there's an env var in between a string? e.g `hello-$NAME`